src/morale.cpp

```diff
@@ -480,7 +480,8 @@ void player_morale::display( double focus_gain )
             const int decimals = ( value - static_cast<int>( value ) != 0.0 ) ? 2 : 0;
             color = ( value > 0.0 ) ? c_green : c_red;
             mvwprintz( w, y, getmaxx( w ) - 8, color, "%+6.*f", decimals, value );
-        } else {
+        } else
+        {
```

Comment:
Just checked: this is the only place in the entire codebase where:

1. the file is in `astyle_whitelist`;
2. the block is a lambda function with a return value;
3. the function body has an `else` clause.

There is a similar place in `src/game_inventory.cpp`, around line 380, that has (1) and (2) but not (3). Giving it an `else` so it does have (3) does not result in the above behaviour.

So there must be other conditions than the three above, like being nested in a specific block...

The very fact that

``` c++
         if( value != 0.0 )
         {
```

is OK in `astyle`'s book raises a flag.